### ISSUE TITLE
I am getting error while Enabling the DNS Cache = true

### ISSUE THREAD

**Issue by ravindrasinghh** [terraform-google-modules/terraform-google-kubernetes-engine#1378]
### TL;DR

I am getting error while Enabling the DNS Cache = true ,I am using below config 
module "gke" { 
source                          = "terraform-google-modules/kubernetes-engine/google//modules/private-cluster"
  version                         = "22.1.0"
  project_id                      = local.project_id
  name                            = ""
  description                     = ""
  regional                        =""
  region                          = ""
  zones                           = ""
  network                         = ""
  network_project_id              = ""
  subnetwork                      = ""
  kubernetes_version              = 
  master_authorized_networks      = ""
  enable_vertical_pod_autoscaling =""
  horizontal_pod_autoscaling      = ""
  http_load_balancing             = ""
  maintenance_start_time          = ""
  maintenance_exclusions          = ""
  maintenance_end_time            = ""
  maintenance_recurrence          = ""
  ip_range_pods                   = ""
  ip_range_services               =""
  node_pools                      = ""
  node_pools_labels               = ""
  node_pools_metadata             = ""
  cluster_autoscaling             =""
  node_pools_taints               = ""
  node_pools_tags                 = ""
  cluster_resource_labels         = ""
  dns_cache                       = true
  enable_private_endpoint         = ""
  enable_private_nodes            = ""
  master_ipv4_cidr_block          = ""
  network_policy                  =""
  remove_default_node_pool        = ""
  filestore_csi_driver            = ""
  enable_shielded_nodes           =""
  enable_binary_authorization     = ""
}

### Terraform Configuration

```hcl
│   on main.tf line 32, in module "gke":
│   32:   dns_cache                       = local.cluster_config.dns_cache
│ 
│ An argument named "dns_cache" is not expected here.
```

### Terraform Version

```sh
1.2.6
```

**Comment by bharathkkb** [comment]
@ravindrasinghh `dns_cache` was promoted to GA modules in version 23. Please try updating the version and feel free to reopen if still an issue.

**Comment by ravindrasinghh** [comment]
yes, Forgot to close this issue.
Thank you for reply @bharathkkb
